### DISCUSSION TITLE
Updating Arbitrary properties not working after compile

### DISCUSSION THREAD

**Discussion by Elkaroui** [tailwindlabs/tailwindcss#7034] [Help]
Hello, I had noticed that when I use CDN I can change the color or other properties like text-[#000000] on the browser but if I remove the CDN and use CLI I lost this ability after compiling the file, is there any way to keep it after compile?

I want to use this feature to update color from Shopify CMS if possible, Thank you~

**Comment by KirankumarAmbati** [comment]
+1

I am trying to do something like `"bg-[#e7e7e7]"` where `#e7e7e7` comes from contentful CMS. But it is not getting rendered correctly.
